README.md

```diff
@@ -8,97 +9,45 @@ __User Clicks "Submit Restart Request"__
 This screen will display __Restart already scheduled.__ if a user has already requested a restart that hasn't completed yet.
 
 ## Installation
-Assumes apache is already setup and running on CentOS and will be secured by Duke Shiboleth/Group Manager.
-Apache should have https setup. 
-
-### Install Shiboleth
-Create /etc/yum.repos.d/duke-el-shib2.repo with the following content:
-```
-[duke-el-shib2]
-name=Shibboleth2
-baseurl=http://download.opensuse.org/repositories/security://shibboleth/CentOS_5/
-enabled=1
-gpgcheck=0
-priority=1
-```
-For CentOS 6 use this URL:
-```
-baseurl=http://download.opensuse.org/repositories/security://shibboleth/CentOS_CentOS-6/
-```
-Change the baseurl for the particular version we will use.
-
-Install package:
-```
-yum install shibboleth.x86_64
-```
-
-### Setup Shiboleth
-Download duke shiboleth config files.
-```
-cd /etc/shiboleth
-wget https://shib.oit.duke.edu/duke-metadata-2-signed.xml
-wget https://shib.oit.duke.edu/idp_signing.crt
-cp <this repo directory>/etc/shibboleth/shibboleth2.xml /etc/shibboleth/shibboleth2.xml
-```
-
-Generate certs.
-```
-cd /etc/shiboleth
-keygen.sh
-```
-Update SP with value from sp-cert.pem
 
+### Setup for website
+Install docker
 
-### Setup Apache
-Turn on UseCanonicalName in /etc/httpd/conf/httpd.conf
+Create a directory named __external__ with the following structure:
 ```
-UseCanonicalName On
+external/
+  shibboleth/
+     sp-cert.pem               # generated by /etc/shibboleth/keygen.sh
+     sp-key.pem                # generated by /etc/shibboleth/keygen.sh
+  ssh/
+     id_rsa                    # ssh setup used to connect to machine that will restart service
+     id_rsa.pub
```

Comment:
The client doesn't need the public key - but if the keypair is generated on this host, it will naturally be there.